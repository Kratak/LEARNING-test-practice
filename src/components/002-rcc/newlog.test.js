import React from 'react'
import {render, cleanup, fireEvent} from 'react-testing-library'
import NewLog from './newlog'

afterEach(cleanup)

test('NewLog testing',()=>{
    const {
        debug , 
        getByTestId, 
        getByText, 
        queryByTestId,
        container} = 
    render(<NewLog />)

    expect(getByTestId('page-title').textContent).toBe('New log')
    expect(queryByTestId('log-form')).toBeTruthy()
    
    expect(container.firstChild).toMatchSnapshot()
})