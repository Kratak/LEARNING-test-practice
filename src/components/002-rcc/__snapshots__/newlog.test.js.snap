// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewLog testing 1`] = `
<h1
  data-testid="page-title"
>
  New log
</h1>
`;
